pkg/nexus/deadline/scheduler/deadline-scheduler_test.go

```diff
@@ -1,37 +1,55 @@
-package deadline
+package deadline_test
 
 import (
+	"bytes"
+	"io"
+	"net/http"
+	"net/url"
+	"testing"
+	"time"
+
 	"github.com/nuclio/nuclio/pkg/nexus/common/models/config"
 	structsCommon "github.com/nuclio/nuclio/pkg/nexus/common/models/structs"
 	common "github.com/nuclio/nuclio/pkg/nexus/common/queue"
 	"github.com/nuclio/nuclio/pkg/nexus/common/scheduler"
-	deadline "github.com/nuclio/nuclio/pkg/nexus/deadline/models"
+	models "github.com/nuclio/nuclio/pkg/nexus/deadline/models"
+	deadline "github.com/nuclio/nuclio/pkg/nexus/deadline/scheduler"
 	"github.com/stretchr/testify/suite"
-	"net/http"
-	"net/url"
-	"testing"
-	"time"
 )
 
 type DeadlineSchedulerTestSuite struct {
 	suite.Suite
-	ds *DeadlineScheduler
+	ds *deadline.DeadlineScheduler
+}
+
+type MockRoundTripper struct{}
+
+func (m *MockRoundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
+	return &http.Response{
+		StatusCode: 200,
+		Body:       io.NopCloser(bytes.NewBufferString("Mocked response")),
+		Header:     make(http.Header),
+	}, nil
```

Comment:
Du wiederholst dich hier. DRY verletzt --> lagere in `nexus>common>test-utils` aus

Reply:
Danke, gute idee